core/src/main/resources/filodb-defaults.conf

```diff
@@ -495,6 +495,9 @@ filodb {
       # 10000 * (1KB per col * 6 cols * 3 chunks) + buffer
       shard-mem-size = 400 MB
 
+      # switch to enable/disable calculation of cardinality in downsample cluster
+      metering-enabled = true
```

Comment:
@amolnayak311 @sherali42 @alextheimer  new to how we manage/add configs. Wondering if we also have to add this config in helm-charts somewhere ?

Reply:
Also, wondering if I should start with this config as false and override for certain tsdb partitions to start with ?